src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1134,3 +1134,30 @@ def wrapper(*args, **kwargs):
     return None
 
   return wrapper
+
+
+def is_tescase_deprecated(platform_id=None):
+  """Whether or not the device is deprecated."""
+  if is_android_cuttlefish(platform_id.upper()):
+    return False
+  if is_android(platform_id.upper()):
+    # FIXME: Handle this import in a cleaner way.
+    from clusterfuzz._internal.platforms import android
+
+    return android.util.is_tescase_deprecated(platform_id)
+
+  return False
+
+def can_testcase_run_on_platform(testcase_platform_id, current_platform_id):
```

Comment:
both of these functions `is_tescase_deprecated` and `can_testcase_run_on_platform` should first check if the environment is not Android, and immediately return `False` since this only applies to Android.

Reply:
Done. Checks added accordingly.